plugin-svn/che-plugin-svn-ext-subversion/src/main/java/org/eclipse/che/ide/ext/svn/server/rest/SubversionService.java

```diff
@@ -560,4 +568,18 @@ private String getRealPath(final String path) throws ServerException {
         }
     }
 
+    @Path("import-source-descriptor")
+    @Produces(MediaType.APPLICATION_JSON)
+    @GET
+    public ImportSourceDescriptor importDescriptor(@Context UriInfo uriInfo,
+                                                   @QueryParam("projectPath") String projectPath) throws ApiException, IOException {
+        final VirtualFile virtualFile = vfsRegistry.getProvider(workspaceId).getMountPoint(true).getVirtualFile(projectPath);
+        if (virtualFile.getChild(".svn") != null) {
+            return DtoFactory.getInstance().createDto(ImportSourceDescriptor.class)
```

Comment:
Use DtoFactory.newDto with static import instead of DtoFactory.getInstance().createDto